ws-client/src/client.rs

```diff
@@ -488,6 +478,17 @@ async fn send_unsubscribe_request(
 	request: RequestMessage,
 ) {
 	if let Err(e) = sender.start_request(request, manager).await {
-		log::error!("send unsubscribe request failed: {:?}", e);
+		log::error!("Send unsubscribe request failed: {:?}", e);
 	}
 }
+
+fn build_unsubscribe_request(
+	manager: &mut RequestManager,
+	req_id: u64,
+	sub_id: SubscriptionId,
+) -> Option<RequestMessage> {
+	let (_, unsub) = manager.remove_subscription(req_id, sub_id.clone())?;
+	manager.reclaim_request_id(req_id);
+	let json_sub_id = jsonrpc::to_value(sub_id).expect("SubscriptionId to JSON is infallible; qed");
+	Some(RequestMessage { method: unsub, params: jsonrpc::Params::Array(vec![json_sub_id]), send_back: None })
```

Comment:
Maybe we should have a `params!` macro to reduce the noise a bit:

`params![json_sub_id]` vs `jsonrpc::Params::Array(vec![json_sub_id])`

Reply:
might go away with `maciejs serde raw value` design :)